projects.yaml

```diff
@@ -806,6 +806,12 @@ projects:
   pypi_id: mkdocs-swagger-ui-tag
   labels: [plugin]
   category: integrations
+- name: Github Releaase Changelog
+  mkdocs_plugin: mkdocs_github_changelog
+  github_id: djpugh/mkdocs_github_changelog
+  pypi_id: mkdocs-github-changelog
+  labels: [plugin]
+  category: integrations
```

Comment:
Hmm isn't " Git repos & info " a better category?

Reply:
I thought as it was GitHub specific it made more sense in integrations

Reply:
OK I see so maybe it's technically wrong to put it into the Git category, but I still think it'd be much much easier to find there.

Reply:
```suggestion
  category: git-info
```